internal/utils/utils.go

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: MIT
+// SPDX-FileCopyrightText: (c) Copyright 2023 Advanced Micro Devices, Inc.
+
+package utils
+
+func NewBoolPointer(v bool) *bool {
+	return &v
+}
```

Comment:
You could use this existing k8s utils package.
https://pkg.go.dev/k8s.io/utils/ptr
Something like `ptr.To[bool](True)`

Reply:
Yep, it's a good idea, thanks.